openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
+    """Enforces correct order of features.
+
+    Alphabetically orders the feature columns. The load column remains the first column
+        and the Horizons column remains the last column. Everything in between is alphabetically sorted.
+
+    Args:
+        df (pd.DataFrame): Dataframe with features.
+
+    Returns:
+        pd.DataFrame with features with all columns in the correct order
+    """
+
+    # Extract first column name
+    first_col_name = df.columns.to_list()[0]  # Most of the time this is load
+
+    # Sort columns
+    columns = list(np.sort(df.columns.to_list()))
+
+    # Remove first column and add to the start
+    columns.remove(first_col_name)
+    col_order = [first_col_name] + columns
+
+    # If "Horzion" column is available add to the end
+    if (
```

Comment:
suggestion, use a single line: `if "Horizon" in columns:`

Reply:
I noticed that we use both "Horizon" and "horizon". I think we should change all column names `lower_snake_case` when possible, right now we mixing many conventions.

Reply:
Good one I changed it to snake_case